README.md

```diff
@@ -27,6 +27,8 @@ To run the example, you can use the following command:
 elixir examples/muxer_isom.exs
 ``` 
 
+You can use `MP4_OUTPUT_FILE` environment variable to set location of the output file (defaults to `example.mp4`).
```

Comment:
Did you consider making it a command line argument instead?
That's fine too, it's just a suggestion

Reply:
I was using convention found in our other plugins. However, I think it's a good idea for future to consider making examples standalone scripts with possibility to pass elements' options as command line arguments 🤔